roles/traefik/defaults/main.yml

```diff
@@ -20,6 +20,7 @@ traefik_log_level: "INFO"
 # find the relevant name and environment variables for your DNS provider at https://go-acme.github.io/lego/dns/
 traefik_dns_provider: cloudflare
 traefik_environment_variables:
+  TZ: "{{ ansible_nas_timezone }}"
```

Comment:
This should be added directly to the `docker_container` definition as with other containers. It'll work how it is, but is not in the right place.

Reply:
Trying to do this:

```
      - TZ: "{{ ansible_nas_timezone }}"
      - "{{ traefik_environment_variables }}"
```
Results in this:

> TASK [traefik : Traefik Docker Container] ******************************************************************************
> fatal: [ansible-nas]: FAILED! => {"changed": false, "msg": "argument env is of type <class 'list'> and we were unable to convert to dict: <class 'list'> cannot be converted to a dict"}